lib/mayaUsd/ufe/Utils.h

```diff
@@ -216,6 +216,13 @@ Ufe::Selection recreateDescendants(const Ufe::Selection& src, const Ufe::Path& f
 MAYAUSD_CORE_PUBLIC
 std::vector<std::string> splitString(const std::string& str, const std::string& separators);
 
+//! Gets the sub strings surrounded by separators.
+MAYAUSD_CORE_PUBLIC
+std::vector<std::string> getSubStrings(
```

Comment:
To me the name of this method makes it sound like this is a generic function, but it really seems kind of specific to the metadata stream, no? Do you think this method is generic enough to warrant it being in Utils.h, or should it just be a helper function in StagesSubject.cpp where it is being used?

Reply:
I agree with Jonathan. The example string you give above ```"'uifolder':,'uisoftmin':0.0, 'uihide':1, 'uiorder':0"``` is very metadata specific. I would move it like Jonathan suggests and also rename it with metadata in name.